apps/apps.c

```diff
@@ -2494,8 +2490,13 @@ BIO *dup_bio_err(int format)
     return b;
 }
 
+/*
+ * The prefix method doesn't have a self-destruct function,
+ * so we murder it instead...
```

Comment:
Reword.  There are other ways of saying this without "murder" ... or self-destruct.

Reply:
I suspected you'd remark on this.

Reply:
Fixed